src/js/components/Controls.js

```diff
@@ -44,6 +46,29 @@ class Controls extends React.PureComponent {
       if (this.mounted && this.state.showBorder === false) {
         this.setState({ showBorder: true });
       }
+
+      const currentScroll = this.$wrapper.offsetTop;
+      const { showDisplayOptions, showFontOptions } = this.props;
+      this.setState(prevState => {
+        const { showControls, lastScrollPos } = prevState;
+
+        if(lastScrollPos > currentScroll) {
+          return {
+            lastScrollPos: currentScroll,
+            showControls: !showControls
+            ? true
+            : showControls
+          };
+        }
+        return {
+          lastScrollPos: currentScroll,
+          showControls: showControls &&
+          !showDisplayOptions &&
+          !showFontOptions ?
+          false :
+          showControls
+        };
+      });
```

Comment:
Are you not using prettier? Please set it up

Reply:
Done. :pray: